### ISSUE TITLE
ToDo

### ISSUE THREAD

**Issue by igomezv** [ChJazhiel/VAE_NBody#5]
- Improve loss function behavior (Jazhiel). 
- Hyperparameter tuning (Isidro).
- Analysis of power spectrum (Jazhiel).
